build_tools/buildkite/github/post_benchmarks_as_pr_comment.py

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env python3
+# Lint as: python3
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Posts benchmark results to GitHub as pull request comments.
```

Comment:
I think it would be nice to also have the prior results posted somehow. Maybe for each PR also run at it's branch point off the main branch? We'd want to cache the results. That can be future work though.

[only here for threading]

Reply:
Yup. Also there might be a way that we can refer to the base commit's previous Buildkite run (that happened for `main`). But all seem extra nice things to have later.